test/description-render-test.js

```diff
@@ -0,0 +1,162 @@
+assert = require('chai').assert;
+
+markdownRenderer = require('../lib/index.js');
+
+describe('Markdown', function() {
+  return describe('#toHtml', function() {
+    
+    it('Parse a plain paragraph', function(done) {
+      var expectedHtml, markdownString;
+      markdownString = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.';
+      expectedHtml = '<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>\n';
+      return markdownRenderer.renderHtml(markdownString, function(error, html) {
+        assert.strictEqual(html, expectedHtml);
+        return done(error);
+      });
+    });
+    it('Parse a bullet list (stars used as bullets)', function(done) {
```

Comment:
Please add an empty line before every `it(...` line. Tough to read it currently at a glance.